plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/commit/CommitPresenter.java

```diff
@@ -218,9 +214,10 @@ public void onCommitClicked() {
   }
 
   private Path[] getFilesToCommitArray() {
-    Path[] filesToCommitArray = new Path[filesToCommit.size()];
-    filesToCommit.forEach(
-        file -> filesToCommitArray[filesToCommit.indexOf(file)] = Path.valueOf(file));
+    List<String> selectedFiles = selectableChangesPanelPresenter.getSelectedFiles();
```

Comment:
remove Array suffix from the method name

Reply:
Return list

Reply:
use List.toArray() method

Reply:
The problem is that `toArray(selectedFiles)` returns `String` array, but the method returns `Path` array